classes/helper.php

```diff
@@ -281,7 +287,7 @@ public static function plain_text_search(string $search, string $table,
                            $tablealias.$columnname,
                            $tablealias.$coursefield as courseid,
                            c.shortname as courseshortname
-                      FROM {".$table."} t
+                      FROM {".$table."} $tablealias
                  LEFT JOIN {course} c ON c.id = t.$coursefield
```

Comment:
Nice find. I think there's one more place 't' should be updated on the following line.